pkg/config/config.go

```diff
@@ -247,8 +247,18 @@ func setModelServerReqEndpoint() string {
 
 // return local path to power model weight
 // e.g., /var/lib/kepler/data/model_weight/acpi_AbsPowerModel.json
-func GetDefaultPowerModelURL(modelOutputType, energySource string) string {
-	return fmt.Sprintf(`/var/lib/kepler/data/model_weight/%s_%sModel.json`, energySource, modelOutputType)
+func GetDefaultPowerModelURL(modelOutputType, energySource, cpuArch string) string {
+	// strip white space or new line from cpuArch
+	cpuArch = strings.TrimSuffix(cpuArch, "\n")
```

Comment:
TrimSpace should remove \n characters and white space, so you can remove this line